cookbook/cache/varnish.rst

```diff
@@ -24,6 +24,21 @@ Remember to configure :ref:`framework.trusted_proxies <reference-framework-trust
 in the Symfony configuration so that Varnish is seen as a trusted proxy and the
 :ref:`X-Forwarded <varnish-x-forwarded-headers>` headers are used.
 
+Varnish, in its default configuration, sends the ``X-Forwarded-For`` header but
+does not filter out the ``Forwarded`` header. If you have access to the Varnish
+configuration file, you can configure Varnish to remove the ``Forwarded``
+header::
+
+.. code-block:: varnish4
+
+    sub vcl_recv {
+        remove req.http.Forwarded;
+    }
+
+If you do not have access to your Varnish configuration, you can instead
+configure Symfony to distrust the ``Forwarded`` header as detailed in
+:doc:`/cookbook/request/load_balancer_reverse_proxy`.
```

Comment:
if you add `.. _cookbook-request-untrust-header:` before the headline on line 60 of the referenced cookbook article, you can use ``:ref:`the cookbook <cookbook-request-untrust-header>` `` here to directly link to the correct section.